src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -2053,3 +2053,285 @@ def cli(self, output=None):
+        #     FSM set allocs                    : 60427           
+        #     FSM set frees                     : 60316           
+        #     Timer allocs                      : 0               
+        # ...OUTPUT OMITTED...             
+
+
+        # MMIF mobility event statistics:
+        #     Event data allocs                 : 354187          
+        #     Event data frees                  : 354187          
+        #     Invalid events                    : 13              
+        #     Event schedule errors             : 0               
+        #     MMIF internal errors:
+    #       IPC failure                     : 0               
+    # ...OUTPUT OMITTED...    
+            
+        # Mobility event statistics:
+        mobility_event_statistics_capture = re.compile(r"^Mobility event statistics:$")
+
+        # Joined as
+        joined_as_capture = re.compile(r"^Joined as$")
+
+        # Delete
+        delete_capture = re.compile(r"^Delete$")
+
+        # Role changes
+        role_changes_capture = re.compile(r"^Role changes$")
+
+        # Roam stats
+        roam_stats_capture = re.compile(r"^Roam stats$")
+
+        # Anchor Request
+        anchor_request_capture = re.compile(r"^Anchor Request$")
+
+        # Handoff Status Received
+        handoff_status_received_capture = re.compile(r"^Handoff Status Received$")
+
+        # Handoff Status Sent
+        handoff_status_sent_capture = re.compile(r"^Handoff Status Sent$")
+
+        # Export Anchor
+        export_anchor_capture = re.compile(r"^Export Anchor$")
+
+        # Response Received             :
+        export_anchor_response_received_capture = re.compile(r"Response Received\s+:")
+
+        # Response Sent             :
+        export_anchor_response_sent_capture = re.compile(r"Response Sent\s+:")
+
+        # MM mobility event statistics:
+        mm_mobility_event_statistics_capture = re.compile(
+            r"^MM mobility event statistics:$"
+        )
+
+        # MMIF mobility event statistics:
+        mmif_mobility_event_statistics_capture = re.compile(
+            r"^MMIF mobility event statistics:$"
+        )
+
+        # MMIF internal errors:
+        mmif_internal_errors_capture = re.compile(r"^MMIF internal errors:$")
+
+        # key : value
+        key_value_capture = re.compile(r"^(?P<key>[\S\s]+\S)\s*:\s+(?P<value>\d+)$")
+
+        wireless_info_obj = {}
+
+        for line in out.splitlines():
+            line = line.strip()
+
+            header_capture_list = [
+                mobility_event_statistics_capture,
+                mm_mobility_event_statistics_capture,
+                mmif_mobility_event_statistics_capture,
+            ]
+
+            for capture in header_capture_list:
+                if capture.match(line):
+                    line_format = line.replace(" ", "_").lower().strip(":")
+                    wireless_info_obj.update({line_format: {}})
```

Comment:
You can use ```setdefault``` to combine the following lines:
```
header_group = wireless_info_obj.setdefault(line_format, {})
header_tracking = "header"

Reply:
Very cool, I appreciate the tip! Updated!